src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -550,5 +553,16 @@ static String getResource(String key) {
                     "MSI Token failure: Failed to acquire access token from IMDS, unexpected error occurred."},
             {"R_MSITokenFailureEndpoint", "MSI Token failure: Failed to acquire token from MSI Endpoint"},
             {"R_propertyNotSupported",
-                    "Microsoft JDBC Driver for SQL Server currently does not support the property: {0}"}};
+                    "Microsoft JDBC Driver for SQL Server currently does not support the property: {0}"},
```

Comment:
Please make sure all the files are formatted.

Reply:
it is